docs/source/upgrade.rst

```diff
@@ -10,6 +10,13 @@ Most changes are carefully designed not to break existing code; however changes
 Chainer v6
 ==========
 
+``Variable`` Became a Subclass of ``collections.abc.Iterable``
+--------------------------------------------------------------
+
+Prior to Chainer v5, :class:`~chainer.Variable` did not implement `__iter__`.
+In Chainer v6, ``issubclass(chainer.Variable, collections.abc.Iterable)`` returns `True`.
+It is recommended to check types by ``isinstance(_, collections.abc.Sequence)`` to distinguish :class:`tuple` and :class:`list` from :class:`~chainer.Variable`.
```

Comment:
From offline discussion:
- `collections.Iterable`, `collections.Sequence` in Python 2.
- Using `Sequence` is not always the best choice.